metecho/api/models.py

```diff
@@ -575,7 +575,11 @@ def finalize_available_task_org_config_names(self, originating_user_id=None):
         self.notify_changed(originating_user_id=originating_user_id)
 
     class Meta:
-        ordering = ("-created_at", "name")
+        ordering = (
+            "status",
+            "-created_at",
+            "name",
+        )
```

Comment:
@wlonk 

1. This is missing a migration, but...
2. Is this valuable to have at all, given the real sorting is now happening in the serializer? That is, we don't actually want to sort by "status" as it does here, so what's the point?

Reply:
Oops, left that in when I didn't mean to, anyway.